modules/plugins/languages/nix.nix

```diff
@@ -90,9 +90,9 @@
       '';
     };
 
-    nixpkgs-fmt = {
-      package = pkgs.nixpkgs-fmt;
-      # Never need to use null-ls for nixpkgs-fmt
+    nixfmt = {
+      package = pkgs.nixfmt-rfc-style;
+      # Never need to use null-ls for nixfmt
```

Comment:
Is this true?

Reply:
It actually isn't. null-ls did support nixfmt with `null_ls.builtins.formatting.nixfmt`, so I used that instead.